assets/js/components/notifications/EntityBannerNotifications.js

```diff
@@ -21,9 +21,19 @@
  */
 import { useFeature } from '../../hooks/useFeature';
 import ZeroDataStateNotifications from './ZeroDataStateNotifications';
+import { ActivationBanner } from '../../modules/analytics-4/components/dashboard';
 
 export default function EntityBannerNotifications() {
 	const zeroDataStatesEnabled = useFeature( 'zeroDataStates' );
+	const ga4ActivationBannerEnabled = useFeature( 'ga4ActivationBanner' );
 
-	return zeroDataStatesEnabled && <ZeroDataStateNotifications />;
+	if ( zeroDataStatesEnabled ) {
+		return <ZeroDataStateNotifications />;
+	}
+
+	if ( ga4ActivationBannerEnabled ) {
+		return <ActivationBanner />;
+	}
+
+	return null;
```

Comment:
Hey @makiost, this actually needs to render both `ZeroDataStateNotifications` and `ActivationBanner` at the same time (similarly to [`BannerNotifications`](https://github.com/google/site-kit-wp/blob/461890cb71cfab47156c999aff3d41f1f43a785e/assets/js/components/notifications/BannerNotifications.js#L73-L74)), albeit conditionally:

```suggestion
	return (
		<Fragment>
			{ zeroDataStatesEnabled && <ZeroDataStateNotifications /> }
			{ ga4ActivationBannerEnabled && <ActivationBanner /> }
		</Fragment>
	);
```